src/Uploader/test/UploadTriggerSpec.tsx

```diff
@@ -1,87 +1,98 @@
+    const { container } = render(<UploadTrigger accept=".jpg" />);
+    // eslint-disable-next-line testing-library/no-node-access, testing-library/no-container
+    assert.ok(container.querySelector('input[accept=".jpg"]'));
   });
 
   it('Should render custom component', () => {
-    const instance = getDOMNode(<UploadTrigger as={'a'} />);
-    assert.equal((instance.querySelector('.rs-uploader-trigger-btn') as HTMLElement).tagName, 'A');
+    const { container } = render(<UploadTrigger as={'a'} />);
+    // eslint-disable-next-line testing-library/no-node-access, testing-library/no-container
+    assert.equal((container.querySelector('.rs-uploader-trigger-btn') as HTMLElement).tagName, 'A');
   });
 
   it('Should call onChange callback', () => {
     const onChange = sinon.spy();
-    const instance = getDOMNode(<UploadTrigger onChange={onChange} />);
-    ReactTestUtils.Simulate.change(instance.querySelector('input') as HTMLElement);
+    const { container } = render(<UploadTrigger onChange={onChange} />);
+    // eslint-disable-next-line testing-library/no-node-access, testing-library/no-container
+    ReactTestUtils.Simulate.change(container.querySelector('input') as HTMLElement);
 
     expect(onChange).to.have.been.calledOnce;
   });
 
   it('Should have a name', () => {
-    const instance = getDOMNode(<UploadTrigger name="file" />);
-    assert.ok(instance.querySelector('input[name="file"]'));
+    const { container } = render(<UploadTrigger name="file" />);
+    console.log(container);
+    // eslint-disable-next-line testing-library/no-node-access, testing-library/no-container
+    assert.ok(container.querySelector('input[name="file"]'));
   });
 
   it('Should have a custom className', () => {
-    const instance = getDOMNode(<UploadTrigger className="custom" />);
-    assert.include(instance.className, 'custom');
+    const { container } = render(<UploadTrigger className="custom" />);
+    expect(container.firstChild).to.have.class('custom');
   });
 
   it('Should have a custom style', () => {
     const fontSize = '12px';
-    const instance = getDOMNode(<UploadTrigger style={{ fontSize }} />);
-    assert.equal((instance.querySelector('button') as HTMLElement).style.fontSize, fontSize);
+    const { container } = render(<UploadTrigger style={{ fontSize }} />);
+    // eslint-disable-next-line testing-library/no-node-access, testing-library/no-container
+    assert.equal((container.querySelector('button') as HTMLElement).style.fontSize, fontSize);
   });
 
   it('Should have a custom className prefix', () => {
-    const instance = getDOMNode(<UploadTrigger classPrefix="custom-prefix" />);
-    assert.ok(instance.className.match(/\bcustom-prefix\b/));
+    const { container } = render(<UploadTrigger classPrefix="custom-prefix" />);
+    // eslint-disable-next-line testing-library/no-node-access, testing-library/no-container
+    assert.ok(container.querySelector('.rs-custom-prefix')?.className.match(/\bcustom-prefix\b/));
```

Comment:
Use `contianer.firstChild`. You may also want to use `expect(...).to.have.class(regex)` assertion :)